webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -311,17 +326,18 @@ const ExperimentDetailsHeader: React.FC<Props> = ({
           menuOptions: [
             {
               disabled: experiment.unmanaged,
+              isLoading: isRunningContinue,
               key: 'continue-trial',
               label: experiment.unmanaged ? (
                 <Tooltip content={UNMANAGED_MESSAGE}>Continue Trial</Tooltip>
               ) : (
                 'Continue Trial'
               ),
-              onClick: ContinueTrialModal.open,
+              onClick: ['random', 'grid'].includes(experiment?.config.searcher.name) ? onClickContinueMultiTrialExp : ContinueTrialModal.open,
```

Comment:
**suggestion:** looks like `['random', 'grid'].includes(experiment?.config.searcher.name)` is done more than once, could have a memoized variable `isSearcherContinuable` or something that does this operation. Or even a more general one that also checks the `RunState` called `isContinuable`